reverse_file.py

```diff
@@ -1 +1,53 @@
+import os
 
+
+def file_length(name: str) -> int:
```

Comment:
The idea was that these functions are provided by the library and you only need to implement `reverse_file_in_memory`. But it won't hurt to exercise them as well

Reply:
a ahaa!  I made it a bit more complicated, sorry for that :D